### PR TITLE
Week3

### PR DESCRIPTION
**Pull request by Roy-M-Moore**

### CODE REVIEW

**Review by veerlevijverberg — APPROVED** [review]
Good job on your vignette! The code snippets provided are clear and concise. It would be beneficial to explain the purpose of each code line or step in more detail, especially for users who are less familiar with R programming. This would help users understand the rationale behind the code and enable them to modify it for their specific needs.

For the "robustness_check()" function, perhaps you could add a description of what the Bayes Factor represents and how to interpret its values (e.g., providing guidance on whether to prefer the alternative or null hypothesis based on the Bayes Factor). Further, it would be useful to outline the intended use cases and advantages of this function.

File: vignettes/BayesianTTestTutorial.Rmd [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+1. Shiny Dashboard tutorial with screenshots.
+
+1. BayesianTTest functions Tutorial.
+
+
+## Shiny Dashboard Tutorial
+
+The shiny dashboard allows the user to perform Bayesian t-tests after they have uploaded a CSV. file of the data. It is not finished yet hence there are no screenshots yet. 
+
+Using the t-test dashboard is really simple, users simply have to upload their data file with one variable in each column. First, they have to select the variable(s) that they want to be part of the design. Then, they will select their desired analyses (e.g., one-sample) and the type of hypothesis (e.g., two tailed). Afterwards, they can specify the prior scale and whether they want a Bayesian robustness check performed.
```
Comment (vincentott) [inline vignettes/BayesianTTestTutorial.Rmd:32]: i recommend to keep the max. number of characters in each line of code to 80.
if you just hit the "enter"-key and split up this line of code, .md should still show it as one paragraph

File: vignettes/BayesianTTestTutorial.Rmd [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+1. "max_r" maximum value of the scale parameter of the Cauchy prior to be plotted
+
+1. "iterations" will determine how precise the plot will be
+
+Here is an example of the function in action:
+
+```{r}
+library(BayesianTTest)
+
+robustness_check(x = sleep[1:10, 1],
```
Comment (vincentott) [inline vignettes/BayesianTTestTutorial.Rmd:64]: As a naive reader of this vignette I am wondering, what the sleep object is.
I would recommend to add one sentence about it above ```{r}.
Even if it should just be a dataset from base R.

File: vignettes/BayesianTTestTutorial.Rmd
Comment (vincentott) [inline vignettes/BayesianTTestTutorial.Rmd:1]: Maybe you could also add one or two more examples just to illustrate the uses of the different arguments for
robustness_check()

**Review by vincentott — APPROVED** [review]

**Pull request merged**